### DISCUSSION TITLE
Creating an admin panel fo Github Pages website

### DISCUSSION THREAD

**Discussion by gecicidegisken** [community/community#22674] [Other Feature Feedback, Questions, & Ideas]
<p>I have a blog-like webpage designed with simply HTML &amp; CSS. In order to upload new posts to my Github-hosted website, I create a new file, with styling, write my post in it, and save the file in my repository. It works perfectly fine. However, if I want to have an admin panel, in order to post easily and manage my site ( like adding tags or comment section) what should I do? What technologies, languages or services I need? I don’t know where to start, so any help would be appreciated. :slight_smile:</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/gecicidegisken">@gecicidegisken</a>, welcome to the GitHub Support Community!</p>
<p>Most sites that use an admin panel to manage content are built using a CMS (Content Management System) such as Wordpress or Drupal. However, these tools are built using PHP, Java, Ruby, or other languages that require server-side processing, which GitHub Pages doesn’t support.</p>
<p>In this instance you should think of <a href="http://GitHub.com">GitHub.com</a> as your content management system and either write your content directly on <a href="http://GitHub.com">GitHub.com</a>, or by cloning your repository locally and writing your content there.</p>
<p>If you do really want to use a CMS to manage your content then I’d recommend searching around for popular CMS tools, though you will also need to find a different hosting provider other than GitHub.</p>

**Comment by gecicidegisken** [comment]
<p>Thank you so much! You saved me from lots of useless effort :slight_smile:  I guess I will continue writing locally, and if I want to go further in the future, I will consider another hosts. Thanks again!</p>
